src/util.test.ts

```diff
@@ -33,27 +34,36 @@ describe("Util", (): void => {
       ): any => {
         callback(error, { stdout, stderr });
       }));
-      const output = await util.execBashCommand(command);
+
+      const output = await util.execBashCommand(command, os);
 
       expect(core.info).nthCalledWith<[string]>(1, command);
+      const shell =
+        os === "win32"
+          ? "C:\\Program Files\\Git\\bin\\bash.exe"
+          : "/usr/bin/bash";
       expect(child_process.exec).lastCalledWith(
         command,
-        { shell: "/usr/bin/bash" },
+        { shell },
         expect.anything()
       );
       return output;
     };
 
+    const mockProcessPlatform = () =>
```

Comment:
This is a misnomer since this is an arbitrary rather than a mock. In keeping with the naming convention of fast-check, it could be named for what it generates (e.g., `platform()`). To avoid confusion, it would also be good to rename the `os` variables to `platform`.